delicious.css

```diff
@@ -0,0 +1,199 @@
+
+body {
+  background: #fff;
+  font-family: Droid Serif, sans-serif;
+  width: 900px;
+  margin: 0 auto;
+  font-size: 2.0rem;
+}
+
+.blog {
+	margin-top: 40px
+	
+}
+
+
+
+/* header stuff */
+
+
+header {
+	height: 80px;
+}
+
+.logo {
+	float: left;
+	margin-right: 4px;
+}
+
+.social {float: right;
```

Comment:
CSS in general needs to be formatted consistently. Conventionally, it will look like:

```
.social {
  float: right;
}
```